mithril-aggregator/src/dependency_injection/builder.rs

```diff
@@ -746,7 +746,7 @@ impl DependenciesBuilder {
     }
 
     async fn build_transaction_parser(&mut self) -> Result<Arc<dyn TransactionParser>> {
-        let transaction_parser = CardanoTransactionParser::default();
+        let transaction_parser = CardanoTransactionParser::new(self.get_logger().await?);
```

Comment:
It would probably be better to put the TODO explaining the defaut value of `allow_unparsable_block` in the builder. 

You could also probably add a configuration parameter for this option and put the default value to `true`. In that case, the TODO could be moved to the `clap` configuration.

Reply:
Done ✅